.travis.yml

```diff
@@ -3,6 +3,7 @@ language: node_js
 node_js:
 - '0.10'
 - '0.12'
+- '4.2'
 - 'iojs-v1.5.1'
```

Comment:
Shall we remove this while we're at it? It seems redundant now that the two projects have merged.